test/dryrun/context_vars/object_cluster_scoped/output_from_cluster.txt

```diff
@@ -0,0 +1,15 @@
+# Diffs:
+v1 Namespace mega-mart:
+--- mega-mart : existing
++++ mega-mart : updated
+@@ -5,10 +5,12 @@
+     city: durham
+   labels:
+     box: big
++    name: mega-mart
++    new-label: durham
+   name: mega-mart
+ spec:
+   finalizers:
```

Comment:
Note: When the kind cluster Diff displays extra unchanged lines, I've created a new txt file to document the differences. The  `+` or `-`  lines between the two `output.txt` and `output_from_cluster.txt` files should be the same: https://github.com/open-cluster-management-io/config-policy-controller/blob/1d9a17651ff30e73c2735a16cfa9f4f268acc83b/test/dryrun/context_vars/object_cluster_scoped/output.txt